lib/bankaccounts/checking_account.rb

```diff
@@ -0,0 +1,47 @@
+module BankAccounts
+
+  class CheckingAccount < Account
+      # attr_accessor :check_count
```

Comment:
This should be uncommented, otherwise you won't be able to read this instance variable or set it to a new value (such as in `reset_checks`).